ledger/src/blockstore.rs

```diff
@@ -3315,6 +3315,54 @@ impl Blockstore {
         );
         Ok(())
     }
+
+    /// Mark a root `slot` as connected, traverse `slot`'s children and update
+    /// the children's connected status if appropriate.
+    ///
+    /// A ledger with a full path of blocks from genesis to the latest root will
+    /// have all of the rooted blocks marked as connected such that new blocks
+    /// could also be connected. However, starting from some root (such as from
+    /// a snapshot) is a valid way to join a cluster. For this case, mark this
+    /// root as connected such that the node that joined midway through can
+    /// have their slots considered connected.
+    pub fn set_and_chain_connected_on_root_and_next_slots(&self, root: Slot) -> Result<()> {
```

Comment:
Correct, children slots of `self`. The use of `next` here is consistent with the terminology we use elsewhere, namely,
https://github.com/solana-labs/solana/blob/17b64005d3d77bdbf2c5f7f2002ce6351d8c59e3/ledger/src/blockstore_meta.rs#L77

For what it is worth, I don't love the use of `next` in this context (we use `parent_slot` so I would like to change to `child_slots`), but for the sake of this PR, I think we should use `next` so that the terminology is consistent.